src/tests/Equinor.ProCoSys.IPO.Test.Common/ReadOnlyTestsBase.cs

```diff
@@ -65,5 +70,13 @@ protected Person AddPerson(IPOContext context, Guid oid, string firstName, strin
             context.SaveChangesAsync().Wait();
             return person;
         }
+
+        protected Person AddSavedFilterToPerson(IPOContext context, Person person)
+        {
+            _savedFilter = new SavedFilter(TestPlant, ProjectName, FilterName, Criteria);
```

Comment:
Person must have more than one saved filter to be able to verfiy correct behaviour. We dont need much updates in the test methods, as we know which filter it should return. But right now we wont catch any errors where the query is wrong, or if the query returns incorrect number of rows.